Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/GeometricFillTool.java

```diff
@@ -168,6 +189,7 @@ public void onSaveInstanceState(Bundle bundle) {
 
 		bundle.putSerializable(BUNDLE_BASE_SHAPE, baseShape);
 		bundle.putSerializable(BUNDLE_SHAPE_DRAW_TYPE, shapeDrawType);
+		bundle.putSerializable(BUNDLE_OUTLINE_WIDTH, shapeOutlineWidth / 2);
```

Comment:
Why `shapeOutlineWidth / 2`?

You retrieve it with `int shapeOutlineWidth = (int) bundle.getSerializable(BUNDLE_OUTLINE_WIDTH);` later on.